lms/templates/student_account/hinted_login.underscore

```diff
@@ -0,0 +1,24 @@
+<div class="hint-wrapper">
```

Comment:
Any objection to me changing this to `wrapper-other-login`, the same class that the other PR is using for the same purpose?